src/comments/dto/create-comment.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { IsArray, IsNotEmpty, IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateCommentDto {
+  @ApiProperty()
+  @IsString()
+  @IsNotEmpty()
+  title!: string;
+
+  @ApiProperty()
+  @IsString()
+  @IsNotEmpty()
+  message!: string;
+
+  @ApiProperty()
+  @IsArray()
```

Comment:
Please check if swagger has managed to reflect the property of Array<String>. If not, add proper type to the `ApiPropertyDecorator`